server/core/runtime/multienv_step_runner.go

```diff
@@ -22,7 +22,7 @@ func (r *MultiEnvStepRunner) Run(ctx command.ProjectContext, command string, pat
 			var sb strings.Builder
 			sb.WriteString("Dynamic environment variables added:\n")
 			for _, item := range envVars {
-				nameValue := strings.Split(item, "=")
+				nameValue := strings.SplitN(item, "=", 2)
 				if len(nameValue) == 2 {
 					envs[nameValue[0]] = nameValue[1]
 					sb.WriteString(nameValue[0])
```

Comment:
I'm also finding that this list of dynamic env vars added is quite noisy, and doesn't really add much value to plans/applies. Would there be any openness to suppressing this?

Reply:
@tapaszto is the original developer, maybe he can chime in why this is here?

Reply:
We use multienv step to add dynamic number of environment variables coming from the result of a webservice call.
Originally we proposed to have a json string which can be parsed safely and handling the errors coming from the runstep result but later the reviewer suggested to have a simple key/value list. We modified accordingly implementing the simplest solution for our requirements and did not assume having "=" in the envvar values.
Here is the PR containing more details of our use case:
https://github.com/runatlantis/atlantis/pull/1793

Reply:
Thanks for the feedback @tapaszto.

@jamengual As it stands, is this a reasonable bug fix to incorporate into the mainline? I think it's probably a pretty common case for env vars to contain `=`, as in my experience they're often base64 encoded.

Reply:
if @tapaszto agrees that this is ok to add and does not change the use case I think it will be ok.

Reply:
and maybe we should update the docs if they do not explain well the use case and or the key=value structure.

Reply:
> if @tapaszto agrees that this is ok to add and does not change the use case I think it will be ok.

It's ok

Reply:
@austinsherron check the docs and see if they will need adjustment due to this change, please.
We can merge it after